build.gradle

```diff
@@ -120,7 +122,7 @@ remapJar.dependsOn(shadowJar)
 task shadowRemapJar(type: RemapJarTask) {
```

Comment:
Might want to look at LambdaBetterGrass buildscript, I ended up removing shadowRemapJar entirely because it was kinda useless now?

Reply:
I see what you mean, got it.